.golangci.yaml

```diff
@@ -54,6 +55,12 @@ linters-settings:
       - name: redefines-builtin-id
   testifylint:
     enable-all: true
+  errcheck:
+    exclude-functions:
+      - (*github.com/spf13/cobra.Command).Help
+      - (*github.com/spf13/cobra.Command).MarkFlagRequired
+      - (*github.com/spf13/pflag.FlagSet).MarkHidden
+      - (*github.com/spf13/pflag.FlagSet).MarkDeprecated
```

Comment:
Should we add the response writer and pterm function we're ignoring here as well?

Reply:
These are things I do want to handle in the future but it is just not ideal right now. Either because it would greatly change function signatures or because the logger isn't present yet.